res/shader/surface.inc.wgsl

```diff
@@ -5,28 +5,39 @@ struct SurfaceConstants {
     terrain_bits: vec4<u32>;     // X_low = shift, X_high = mask
 };
 
+struct TerrainData {
+    height: array<u32>;
+};
+
+struct Surface {
+    low_alt: f32;
+    high_alt: f32;
+    delta: f32;
+    low_type: u32;
+    high_type: u32;
+    tex_coord: vec2<f32>;
+    is_shadowed: bool;
+};
+
+struct StorageIndex {
+    index: u32;
+    mask: u32;
+    shift: u32;
+};
+
 @group(1) @binding(0) var<uniform> u_Surface: SurfaceConstants;
 
 @group(1) @binding(2) var t_Height: texture_2d<f32>;
```

Comment:
let's remove the `height` and `meta` bindings since we aren't using them any more